pkg/ibmcsidriver/controller_helper.go

```diff
@@ -296,6 +335,59 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 	return volume, nil
 }
 
+// setSecurityGroupList
+func setSecurityGroupList(volume *provider.Volume, value string) {
+	securityGroupstr := strings.TrimSpace(value)
+	securityGroupList := strings.Split(securityGroupstr, ",")
+	var securityGroups []provider.SecurityGroup
+	for _, securityGroup := range securityGroupList {
+		securityGroups = append(securityGroups, provider.SecurityGroup{ID: securityGroup})
+	}
+	volume.VPCVolume.SecurityGroups = &securityGroups
+}
+
+// setISENIEnabled
+func setISENIEnabled(volume *provider.Volume, key string, value string) error {
+	var err error
+	if value != TrueStr && value != FalseStr {
+		err = fmt.Errorf("'<%v>' is invalid, value of '%s' should be [true|false]", value, key)
```

Comment:
can we accept true, True, false and False values

Reply:
Done